wms_connector/models/attachment_synchronize_task.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Akretion
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AttachmentSynchronizeTask(models.Model):
+    _inherit = "attachment.synchronize.task"
+
+    file_type = fields.Selection(
+        selection_add=[
+            ("export", "Export"),
```

Comment:
nitpicking : `export` should be exist in `attachment_synchonize` module and not be here.
Can be solved later